behavior/MouseTracking/get_mouse_XY_pos.m

```diff
@@ -2,24 +2,33 @@
 % Extract the X,Y positions ( centroids ) of the mouse from the behavior
 %   video ( movie ), with live monitor option (specify 'displayTracking')
 %  
+% Example uses:
+%     Normal mode:
+%       centroids = get_mouse_XY_pos('c9m7d08_ti2-sub.mp4');
+%   Live monitor mode:	
+%         centroids = get_mouse_XY_pos('c9m7d08_ti2-sub.mp4','displayTracking');
+%     Trial-aware mode:
+%       centroids = get_mouse_XY_pos('c9m7d08_ti2-sub.mp4','c9m7d08_cr_ti2.txt');
+% 
 % Input:
 %   - movie: Behavior video (m4v); movie should be cropped (wavy curtain
 %   leads to fall centroids (haven't found another way to fix this)
-%   - varargin: 'displayTracking' displays side-by-side of original movie and
-%   processing movie, along with centroid on each. blue * means good
-%   centroid value, red * means could not find mouse, using previous
-%   centroid value
+%   - varargin:
+%     'displayTracking': Displays side-by-side of original movie and
+%           processing movie, along with centroid on each. blue * means good
```

Comment:
Nah, it just took me a few seconds to realize what you had meant here (and why a "side-by-side" is needed when only a single movie is going into the function).